pennylane/measurements/measurements.py

```diff
@@ -127,27 +127,23 @@ class MeasurementProcess:
         log_base (float): Base for the logarithm.
     """
 
-    # pylint: disable=too-few-public-methods
     # pylint: disable=too-many-arguments
-
     def __init__(
         self,
         return_type: ObservableReturnTypes,
         obs: Operator = None,
         wires=None,
         eigvals=None,
         id=None,
-        log_base=None,
     ):
         self.return_type = return_type
         self.obs = obs
         self.id = id
-        self.log_base = log_base
 
         if wires is not None and obs is not None:
             raise ValueError("Cannot set the wires if an observable is provided.")
 
-        self._wires = wires or Wires([])
+        self._wires = wires
```

Comment:
We keep `self._wires` as the input wires. This will help us simplify the `__copy__` method.

When calling `MeasurementProcess.wires` (public one) we return `Wires([])` if `self._wires` is `None`.